### PR TITLE
LPS-129018 Increase MissingRequiredFieldsModal tests

### PR DESCRIPTION
**Pull request by ramonmsp**

### CODE REVIEW

**Comment by ramonmsp** [comment]
ci:stop

**Comment by ramonmsp** [comment]
cc: @interaminense

**Comment by ramonmsp** [comment]
ci:test:sf

**Comment by ramonmsp** [comment]
ci:test:relevant

File: modules/dxp/apps/app-builder-workflow/app-builder-workflow-web/test/pages/apps/edit/MissingRequiredFieldsModal.es.js [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+const MissingRequiredFieldsModalWrapper = ({fieldType}) => {
+
+    const [deployModalVisible, setDeployModalVisible] = useState(true);
+    
+    const onCloseMissingRequiredFieldsModal = () =>
+        setDeployModalVisible(false);
+    
+    return (
+        <>
+            <MissingRequiredFieldsModal
```
Comment (kevenleone) [inline modules/dxp/apps/app-builder-workflow/app-builder-workflow-web/test/pages/apps/edit/MissingRequiredFieldsModal.es.js:27]: @ramonmsp , this fragment could be removed?

File: modules/dxp/apps/app-builder-workflow/app-builder-workflow-web/test/pages/apps/edit/MissingRequiredFieldsModal.es.js [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+ * contacting Liferay, Inc. See the License for the specific language governing
+ * permissions and limitations under the License, including but not limited to
+ * distribution rights of the Software.
+ */
+
+ import '@testing-library/jest-dom/extend-expect';
+ import {act, fireEvent, render} from '@testing-library/react';
+ import React, {useState} from 'react';
+
+ import MissingRequiredFieldsModal from '../../../../src/main/resources/META-INF/resources/js/pages/apps/edit/MissingRequiredFieldsModal.es';
```
Comment (kevenleone) [inline modules/dxp/apps/app-builder-workflow/app-builder-workflow-web/test/pages/apps/edit/MissingRequiredFieldsModal.es.js:16]: SF is missing on this file, you can run ```yarn format```

**Review by kevenleone — CHANGES_REQUESTED** [review]
I left 2 comments about SF, the rest is really good !

**Review by naraandrade — CHANGES_REQUESTED** [review]

**Comment by ramonmsp** [comment]
ci:test:sf

**Comment by kevenleone** [comment]
LGTM 🔥

**Comment by ramonmsp** [comment]
ci:test:relevant

**Review by naraandrade — APPROVED** [review]

**Comment by kevenleone** [comment]
ci:test:relevant

**Comment by kevenleone** [comment]
ci:test:relevant

**Comment by naraandrade** [comment]
ci:forward

**Comment by naraandrade** [comment]
ci:test:relevant

**Comment by kevenleone** [comment]
ci:forward

**Pull request closed without merging**